plugins/main/bridge/bridge.go

```diff
@@ -386,6 +454,14 @@ func setupVeth(netns ns.NetNS, br *netlink.Bridge, ifName string, mtu int, hairp
 		}
 	}
 
+	// S1031: unnecessary nil check around range (gosimple)
```

Comment:
no need to add this comment I'd say

Reply:
I think this is to appease the linter.

Reply:
I just do a note to make sure no one will add a nil check there again.
I could remove it later

Reply:
ok, i removed it.